### DISCUSSION TITLE
Repeating Panels with Repeating Rows

### DISCUSSION THREAD

**Discussion by milinpatel13298** [grafana/grafana#56824] [Dashboards, Panels & Visualizations]
Hi,
I wanted to know if there was any possibility/talks about adding a feature to access the row variable name, if repeating rows are used?
For example, if there is a dashboard variable, `env`, based on which rows are repeated; is it possible to create a repeating panel in each row using that `env` value, so that, the panels can be repeated for another variable, say `server`,  that is dependent on the `env` variable.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
